src/utils/submission.rs

```diff
@@ -16,12 +16,14 @@ pub async fn create(
             id: None,
             lang,
             code,
-            problem_id: problem_id.to_string(),
+            problem: ("problem", problem).into(),
             status: crate::models::submission::Status::InQueue,
```

Comment:
Consider `use crate::models::submission::Status` and replace with `Status::InQueue`.